components/uniset/src/uniset.rs

```diff
@@ -183,6 +188,93 @@ impl UnicodeSet {
             None => false,
         }
     }
+
+    /// Check if the calling Unicodeset contains all the characters of the given UnicodeSet
+    ///
+    /// # Example:
+    ///
+    /// ```
+    /// use icu_unicodeset::UnicodeSet;
+    /// let example_list = vec![65, 70, 85, 91]; // A - E, U - Z
+    /// let example = UnicodeSet::from_inversion_list(example_list).unwrap();
+    /// let a_to_d = UnicodeSet::from_inversion_list(vec![65, 69]).unwrap();
+    /// let f_to_t = UnicodeSet::from_inversion_list(vec![70, 85]).unwrap();
+    /// let r_to_x = UnicodeSet::from_inversion_list(vec![82, 88]).unwrap();
+    /// assert!(example.contains_set(&a_to_d)); // contains all
+    /// assert!(!example.contains_set(&f_to_t)); // contains none
+    /// assert!(!example.contains_set(&r_to_x)); // contains some
+    /// ```
+    pub fn contains_set(&self, set: &UnicodeSet) -> bool {
+        if set.as_inversion_list().len() > self.inv_list.len() {
```

Comment:
Why?

`A–B + C–D` has 4 elements in the inversion list, but it is contained within `A–Z`, which has only 2 elements in the inversion list.

Reply:
this was meant to be `.size()` but I accidentally changed it to `.len()`

agreed and fixed